test/README.md

```diff
@@ -17,7 +17,8 @@ GitHub with the `autocrlf` git config flag set to true.
 | ------------------- | --------------- | --------------- |
 | `abort`             | Yes             | Tests for when the  ``` --abort-on-uncaught-exception ``` flag is used. |
 | `addons`            | Yes             | Tests for [addon](https://nodejs.org/api/addons.html) functionality along with some tests that require an addon to function properly. |
-| `addons-napi`        | Yes            | Tests for [n-api](https://nodejs.org/api/n-api.html) functionality. |
+| `js-native-api`      | Yes            | Tests for Node.js-agnostic [n-api](https://nodejs.org/api/n-api.html) functionality. |
```

Comment:
It seems the table is sorted alphabetically by the first column and these 2 new entries are out of the order now.